src/components/common/Header.jsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.